nemo_rl/models/generation/vllm/vllm_worker_async.py

```
@@ -46,10 +130,247 @@ def _create_engine(self, llm_kwargs: dict[str, Any]) -> None:
+                    content=generator.model_dump(), status_code=generator.code
+                )
+            elif isinstance(generator, TokenizeResponse):
+                return JSONResponse(content=generator.model_dump())
+
+        ########################################
+        # Server spinup
+        ########################################
+
+        config = uvicorn.Config(
+            app,
+            host="0.0.0.0",
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)